kong/init.lua

```diff
@@ -746,6 +764,18 @@ function Kong.init_worker()
 
   kong.db:set_events_handler(worker_events)
 
+  if process.type() == "privileged agent" then
+    if kong.clustering then
+      kong.clustering:init_worker()
+    end
+
+    -- because of resurrection, objects with finalizers are collected in two phases
+    collectgarbage()
+    collectgarbage()
```

Comment:
Same reason, these two `collectgarbage()` could be wrapped into a function.

Reply:
collectgarbages were removed.